fhir-core/src/main/java/com/ibm/fhir/exception/FHIRException.java

```diff
@@ -85,8 +85,8 @@ public String getUniqueId() {
     }
 
     @Override
-    public String getMessage() {
-        String superMsg = super.getMessage();
+    public String toString() {
```

Comment:
by overwriting toString instead of getMessage, it becomes possible to call getMessage to get just the message (without the probeId).  full ramifications of this change are not yet known/addressed.